Airports/EGPH/SMR/Labels.txt

```diff
@@ -58,10 +58,18 @@
 "9" N055.56.57.290 W003.21.51.101 standHold
 "10" N055.56.58.139 W003.21.48.609 standHold
 "11" N055.56.58.901 W003.21.46.115 standHold
-"12" N055.56.58.920 W003.21.43.480W standHold
-"14" N055.57.00.064 W003.21.44.710 standHold
-"16" N055.56.57.192 W003.21.36.091 standHold
-"18" N055.56.55.716 W003.21.34.178 standHold
+"12" N055.56.58.920 W003.21.43.480 standHold
+"14" N055.56.59.630 W003.21.43.740 standHold
+;"15A" N055.57.00.400 W003.21.37.440 standHold ;Not shown for clarity
+"15" N055.57.00.160 W003.21.39.880 standHold
+;"15B" N055.57.00.990 W003.21.40.020 standHold ;Not shown for clarity
+;"16A" N055.56.57.680 W003.21.36.260 standHold ;Not shown for clarity
+"16" N055.56.58.120 W003.21.38.190 standHold
+;"16B" N055.56.58.700 W003.21.38.090 standHold ;Not shown for clarity
+"17" N055.56.56.040 W003.21.37.090 standHold
+"18" N055.56.55.260 W003.21.36.100 standHold
+"19" N055.56.54.500 W003.21.34.280 standHold
+;"20" N055.56.54.270 W003.21.31.620 standHold ;Not shown for clarity
 "21" N055.56.54.840 W003.21.31.607 standHold
 "22" N055.56.54.999 W003.21.28.641 standHold
 "23" N055.56.53.617 W003.21.27.923 standHold
```

Comment:
Stand 24 missing.
![default](https://user-images.githubusercontent.com/18355854/29998228-723128a0-902e-11e7-8be8-54f0a2e36aa6.png)

Reply:
Again, this is the same as the 2A, 3A, 4A situation.